src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -507,11 +541,11 @@ export default class geTemplateBuilderFormFields extends LightningElement {
     handleDeleteFormElement(event) {
         const fieldMapping = TemplateBuilderService.fieldMappingByDevName[event.detail.fieldName];
```

Comment:
We can remove `fieldMapping` as it's no longer used.

Reply:
fixed!